src/__stories__/select-story.tsx

```diff
@@ -7,14 +7,21 @@ export default {
     component: Select,
 };
 
-const fruitOptions = fruitEntries.map(([text, value]) => ({text, value}));
+const fruitOptions = [
+    ...fruitEntries,
+    ...fruitEntries.map(([text, value]) => [text + 1, value + 1]),
+    ...fruitEntries.map(([text, value]) => [text + 2, value + 2]),
+    ...fruitEntries.map(([text, value]) => [text + 3, value + 3]),
+    ...fruitEntries.map(([text, value]) => [text + 4, value + 4]),
+    ...fruitEntries.map(([text, value]) => [text + 5, value + 5]),
```

Comment:
I'd just add more fruits to the array. here you have some (generated by Copilot):
```js
    ['Pineapple', 'pineapple'],
    ['Strawberry', 'strawberry'],
    ['Watermelon', 'watermelon'],
    ['Grapes', 'grapes'],
    ['Mango', 'mango'],
    ['Peach', 'peach'],
    ['Cherry', 'cherry'],
    ['Raspberry', 'raspberry'],
    ['Blueberry', 'blueberry'],
    ['Kiwi', 'kiwi'],
    ['Pomegranate', 'pomegranate'],
    ['Papaya', 'papaya'],
    ['Coconut', 'coconut'],
    ['Avocado', 'avocado'],
    ['Cantaloupe', 'cantaloupe'],
    ['Mandarin', 'mandarin'],
    ['Melon', 'melon'],
    ['Tangerine', 'tangerine'],
    ['Tomato', 'tomato'],
```